packages/devtools_app/test/test_data/network.dart

```diff
@@ -249,11 +249,9 @@ final Map<String, dynamic> httpPostJson = {
   'responseBody': httpPostResponseBodyData,
 };
 final httpPostRequestBodyData = [
-  ...[32, 32, 32, 32, 123, 10, 32, 32, 32, 32, 32, 32, 116, 105, 116, 108],
-  ...[101, 58, 32, 39, 102, 111, 111, 39, 44, 10, 32, 32, 32, 32, 32, 32],
-  ...[98, 111, 100, 121, 58, 32, 39, 98, 97, 114, 39, 44, 10, 32, 32, 32],
-  ...[32, 32, 32, 117, 115, 101, 114, 73, 100, 58, 32, 49, 44, 10, 32, 32],
-  ...[32, 32, 125, 10, 32, 32, 32, 32],
+  ...[123, 10, 32, 34, 116, 105, 116, 108, 101, 34, 58, 32, 34, 102, 111, 111],
```

Comment:
Why did this data need to be changed?

Reply:
The problem is that the encoded request body contains single quotes that cannot be decoded through the jsonDecode method. I replaced the single quotes with double ones
Before:
`    {
      title: 'foo',
      body: 'bar',
      userId: 1,
    }
    `
After:
`{
 "title": "foo", "body": "bar", "userId": 1
}
 `

Reply:
I think this change is breaking some other tests.

Reply:
I have studied this issue.  Single quotes for names and string literals are supported only in JSON 5 and dart convert's json decode does not support JSON 5, therefore the request body with single quotes is invalid. Unfortunately, this change really breaks some tests, specifically curl_command_test.dart, I suggest making json valid for this test and modifying it

Reply:
Yeah, that seems like a reasonable change to make as long as the tests still pass :-)